docs/src/components/SubscriptionForm.astro

```diff
@@ -85,6 +85,13 @@ const { data } = await getEntry('i18n', lang ?? 'en');
       text-decoration: none;
       font-size: var(--sl-text-sm);
       background: var(--sl-color-text-accent);
+      transition: var(--button-transition);
+
+      &:hover {
+        top: unset;
```

Comment:
`top: unset` because the current implementation is from a library i guess, so i couldn't find the styling in this project. Moving the button up with `top` did work but the transition doesn't work for the `top` attribute, now the button moves up through `transform: translateY(-1px)`.